crates/red_knot_python_semantic/resources/mdtest/attributes.md

```diff
@@ -1131,21 +1222,33 @@ def _(flag: bool):
     def inner3(a_and_b: Intersection[A3, B3]):
         # error: [possibly-unbound-attribute]
         reveal_type(a_and_b.x)  # revealed: P & Q
+
+        # error: [possibly-unbound-attribute]
```

Comment:
"possibly unbound" is a weird phrasing for an attribute we are assigning to, but we can revisit this in future.